sample_json/alpaca/account/account.json

```diff
@@ -1,28 +1,110 @@
 {
-  "account_blocked": false,
-  "account_number": "010203ABCD",
-  "buying_power": "262113.632",
-  "cash": "-23140.2",
-  "created_at": "2019-06-12T22:47:07.99658Z",
-  "currency": "USD",
-  "daytrade_count": 0,
-  "daytrading_buying_power": "262113.632",
-  "equity": "103820.56",
-  "id": "e6fe16f3-64a4-4921-8928-cadf02f92f98",
-  "initial_margin": "63480.38",
-  "last_equity": "103529.24",
-  "last_maintenance_margin": "38000.832",
-  "long_market_value": "126960.76",
-  "maintenance_margin": "38088.228",
-  "multiplier": "4",
-  "pattern_day_trader": false,
-  "portfolio_value": "103820.56",
-  "regt_buying_power": "80680.36",
-  "short_market_value": "0",
-  "shorting_enabled": true,
-  "sma": "0",
-  "status": "ACTIVE",
-  "trade_suspended_by_user": false,
-  "trading_blocked": false,
-  "transfers_blocked": false
-}
\ No newline at end of file
+  "type": "object",
```

Comment:
For future reference -- how were these schemas created?  Is it basically just looking at the alpaca documentation and creating these files?  Or is there another step?

Reply:
For generating the schemas, I created a simple DOM parser using XPaths which allow you to grab XML tags really easily without digging deep through a bunch of nodes and such. So you can just copy and paste in Alpaca or Polygon Documentation HTML and automatically generate the schemas (adding a few adjustments as needed, such as inheritance). The util class is located [here](https://github.com/mainstringargs/alpaca-java/pull/21/files#diff-e04eea01e5149706e1ef7e5cce402d4e).

Reply:
Very slick